clients/bazel-module-registry/src/funTest/kotlin/BazelModuleRegistryClientFunTest.kt

```diff
@@ -46,4 +46,28 @@ class BazelModuleRegistryClientFunTest : WordSpec({
             sourceInfo.integrity shouldBe "sha256-7t5x8oNxvzmqabRd4jsynTchQBbiBVJps7Xnz9QLWfU="
         }
     }
+
```

Comment:
Commit: Can you help me understand why we need support for both <7.2.0 and >= 7.2.0?

Reply:
I added a sentence.